### PR TITLE
Notification 1.1

### PR DESCRIPTION
**Pull request by simonkorz**

### CODE REVIEW

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/database/operations/NotificationOperations.java [hunk 1]
```diff
@@ -226,17 +200,58 @@ public Notification updateNotification(int id, Notification notification) {
...
      */
-    public Result<Record> runReadOnlySQL(String sql) {
-        return readOnlyCreate.fetch(sql);
+    public List<NotificationTokenRecord> diffTokenRecords(Map<Integer, NotificationTokenRecord> newTokenRecords, int notificationId) {
+        if (newTokenRecords != null && !newTokenRecords.isEmpty()) {
+            List<Query> deletionQueries = new ArrayList<>();
+
+            try (Stream<NotificationTokenRecord> recordStream = create.selectFrom(NOTIFICATION_TOKEN)
+                    .where(NOTIFICATION_TOKEN.NOTIFICATION.eq(notificationId))
+                    .fetchLazy().stream()) {
```
Comment (LeanderK) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/database/operations/NotificationOperations.java]: .fetchLazy().stream() is the same as calling directly .stream(), but i think .stream() closes correctly

**Comment by LeanderK** [comment]
push @phimonk

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/database/operations/NotificationOperations.java (hunk 1 above)
Comment (simonkorz) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/database/operations/NotificationOperations.java]: I thought the same and tried it, but it didn't seem to work then.
Now I tried again and everythings fine. Maybe I was already to tired...

**Comment by simonkorz** [comment]
push @LeanderK ready to merge

**Pull request merged**